frontend/public/components/_sysevent-stream.scss

```diff
@@ -104,12 +104,27 @@ $color-dark-border: #ddd;
 }
 
 .co-sysevent__message {
+  cursor: help;
   margin-top: 10px;
   overflow: hidden;
   text-overflow: ellipsis;
   word-wrap: break-word;
-  height: 45px;
+  height: 40px;
 }
+.co-sysevent__message-fade {
```

Comment:
Any reason not to just add this to `co-sysevent__message` ? Otherwise I think this should be `.co-sysevent__message--fade` since it seems like a BEM modifier